appyters/dimensionality_reduction_visualization/dimensionality_reduction_visualization.ipynb

```diff
@@ -84,8 +84,7 @@
     "%%appyter code_exec\n",
     "{% set rnaseq_data_filename = FileField(\n",
     "    name='rnaseq_data_filename', \n",
-    "    label='RNA-seq data file (.csv or .txt)', \n",
-    "    default='Papillary+adenocarcinoma,+NOS_data.csv',\n",
```

Comment:
The error is because you no longer have a default file for testing

Reply:
Thanks for pointing it out!